pandas/_typing.py

```diff
@@ -23,6 +23,7 @@
     Optional,
     Protocol,
     Type as type_t,
+    TypeAlias,
```

Comment:
This is not supported in python 3.9, so you should import it from `typing_extensions`